.github/workflows/ci.yml

```diff
@@ -25,16 +25,24 @@ jobs:
         run: npm install
 
       - name: Lint Files
-        run: node Makefile lint
+        uses: trunk-io/trunk-action@v1
+        with:
+
+          # Run on everything except the docs folder.
+          arguments: --ignore=docs/**
```

Comment:
Does this lint all files or only files changed in the PR/push? Per the docs (https://github.com/trunk-io/trunk-action?tab=readme-ov-file#running-trunk-check-on-all-files), it seems that the default is only files changes, but as noted in https://github.com/eslint/eslint/pull/18643#issuecomment-2250462479 we'd like to always lint all files.

Reply:
Split eslint into it's own step so that it will lint all files. Thanks.